api/buildings/static/js/admin/here_map.js

```diff
@@ -0,0 +1,167 @@
+(function($) {
+  // We'll insert the map after this element:
+  var prev_el_selector = "#id_lng";
+
+  // The input elements we'll put lat/lon into and use
+  // to set the map's initial lat/lon.
+  var lat_input_selector = "#id_lat";
```

Comment:
Why not store the references to the elements themselves here, instead of their ids ? Do they change during the live of the app ?

Reply:
Based on [this SO answer](https://stackoverflow.com/questions/24053838/store-jquery-selector-in-variable) and [the official jQuery documentation](https://learn.jquery.com/using-jquery-core/document-ready/), I understand that was I to "store" the jQuery selectors as you suggest, I would store an empty reference in each variable, as the DOM is not yet rendered (not a jQuery expert, so I'm only trying to explain this to myself as well).

Reply:
I'm presuming that what you [wrote here](https://github.com/code4romania/seismic-risc/pull/684/files/49c3a09020973f1ed21d1bd2101cb7413d0a1f25#diff-214e4cc4a6f9ff58d40482c14b1b6a14439ec65f13380beea39d4117d5c011f3R164-R166) will take care of the document being ready when calling init, but if you observe it to be empty at this point, it's nothing wrong with your approach, so we could leave it as is.